scripts/Interface/ui/drill/model/specifications.py

```diff
@@ -0,0 +1,210 @@
+    COLUMNS = {
+            SANS_ACQ: [
+                "SampleRuns",
+                "SampleTransmissionRuns",
+                "AbsorberRuns",
+                "BeamRuns",
+                "FluxRuns",
+                "ContainerRuns",
+                "ContainerTransmissionRuns",
+                "TransmissionAbsorberRuns",
+                "TransmissionBeamRuns",
+                "MaskFiles",
+                "ReferenceFiles",
+                "OutputWorkspace",
+                "WedgeWorkspace",
+                "SampleThickness",
+                "CustomOptions"
+                ],
+            REFL_POL: [
+                "Run00",
+                "Run01",
+                "Run10",
+                "Run11",
+                "DirectRun",
+                "OutputWorkspace",
+                "AngleOption",
+                "Theta",
+                "SummationType",
+                "CustomOptions",
+                "GlobalScaleFactor",
+                "UseManualScaleFactors",
+                "ManualScaleFactors"
+                ],
+            REFL_NPOL: [
+                "Run",
+                "DirectRun",
+                "OutputWorkspace",
+                "GlobalScaleFactor",
+                "UseManualScaleFactors",
+                "ManualScaleFactors",
+                "AngleOption",
+                "Theta",
+                "SummationType",
+                "CustomOptions"
+                ]
+            }
+
+    # algo name for each acquisition mode
+    ALGORITHM = {
+            SANS_ACQ:   "SANSILLAutoProcess",
+            REFL_POL:   "ReflectometryILLAutoProcess",
+            REFL_NPOL:  "ReflectometryILLAutoProcess"
+            }
+
+    # ideal number of threads for each acquisition mode (optional).
+    # If not provided, Qt will decide
+    THREADS_NUMBER = {
+            }
+
+    # settings for each acquisition mode
+    SETTINGS = {
+            SANS_ACQ : {
+                "ThetaDependent": True,
+                "SensitivityMaps": "",
+                "DefaultMaskFile": "",
+                "NormaliseBy": "Timer",
+                "SampleThickness": "0.1",
+                "BeamRadius": "0.05",
+                "WaterCrossSection": "1",
+                "OutputType": "I(Q)",
+                "CalculateResolution": "None",
+                "DefaultQBinning": "PixelSizeBased",
+                "BinningFactor": "1",
+                "OutputBinning": "",
+                "NPixelDivision": "1",
+                "NumberOfWedges": "0",
+                "WedgeAngle": "30",
+                "WedgeOffset": "0",
+                "AsymmetricWedges": False,
+                "MaxQxy": "0",
+                "DeltaQ": "0",
+                "IQxQyLogBinning": False
+                },
+            REFL_POL : {
+                "PolarizationEfficiencyFile": "",
+                "DirectFlatBackground": "Background Average",
+                "ReflFlatBackground": "Background Constant Fit",
```

Comment:
This as well should be `Background Average`. However this option is introduced in another branch, so it won't work yet on master.

Reply:
Btw, there is a flag in the `ReflectometryILLAutoProcess` for `Polarized` or not, so this should be passed respectively based on the choice of the mode in the interface.

Reply:
Also, `CustomOptions` is not the last column for polarized refl.

Reply:
* I set bask to "Background Constant Fit" to be compatible with master. I will change everything when the corresponding PR is merged.
* I added a FLAGS dictionary in `configuration.py` to set parameters that should not appear in the table or global settings. The reflectometry flag you mention is in.
* `CustomOptions` is now the last column